identity/app/views/fragments/newsletterSwitch.scala.html

```diff
@@ -26,7 +26,7 @@
 }
 
 @newsletterListId = @{
-    if (IdentityGdprMarketingConsentSwitch.isSwitchedOn)
+    if (IdentityCollectGdprCompliantConsentsSwitch.isSwitchedOn)
         newsletter.listId.toString
     else
         newsletter.listIdV1.toString
```

Comment:
I think this is an example of where it would be nice to remove this switch - I don't think we can go back to V1 now so let's remove this references to V1 newsletters along with the switch?